src/components/OptionsList.js

```diff
@@ -234,6 +234,11 @@ class OptionsList extends Component {
                     ref={this.props.innerRef}
                     indicatorStyle="white"
                     keyboardShouldPersistTaps="always"
+
+                    // Does not work on Android
```

Comment:
Hmm, I feel like this isn't really how we should do this. If there is a property that doesn't work on Android, then I think we should have an android component for `SectionList` which doesn't use this property (using the `index.android.js` file extension.

Reply:
OK, thank you for explaining that. I think it's OK to keep it like it is, but maybe change the comment to be a little more clear. Something like, "Both of these props are needed to ensure the keyboard is dismissed on all platforms"